Samples/WinMLSamplesGallery/WinMLSamplesGallery/Samples/AdapterSelection/AdapterSelection.xaml

```diff
@@ -0,0 +1,51 @@
+<Page
+    x:Class="WinMLSamplesGallery.Samples.AdapterSelection"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:local_controls="using:WinMLSamplesGallery.Controls"
+    xmlns:local_samples="using:WinMLSamplesGallery.Samples"
+    mc:Ignorable="d"
+    FontFamily="Arial">
+
+    <Page.Resources>
+        <x:String x:Key="SourceGitHubURL">
+            https://github.com/microsoft/Windows-Machine-Learning/blob/user/numform/adapter_selection/Samples/WinMLSamplesGallery/WinMLSamplesGalleryNative/AdapterList.cpp
+        </x:String>
+    </Page.Resources>
+
+    <StackPanel Margin="0,20,0,0">
+        <Border BorderBrush="Gray" BorderThickness="2" CornerRadius="5">
+            <ListView x:Name="AdapterListView" SelectedIndex="0" SelectionChanged="ChangeAdapter">
+                <ListView.ItemContainerStyle>
+                    <Style TargetType="ListViewItem">
+                        <Setter Property="BorderBrush" Value="LightGray" />
+                        <Setter Property="BorderThickness" Value="0,0,0,1" />
+                    </Style>
+                </ListView.ItemContainerStyle>
+            </ListView>
+        </Border>
+        <ComboBox x:Name="CodeSnippetComboBox" SelectedIndex="0" BorderBrush="#dbdbdb" Header="" Width="70" FontSize="18" Margin="0,20,0,0"
+                  Typography.Capitals="AllSmallCaps" Typography.StylisticSet4="True"
+                  HorizontalAlignment="Right">
+            <x:String>C#</x:String>
+        </ComboBox>
+        <Grid x:Name="CodeSnippet" Background="#F6F6F6" BorderBrush="#dbdbdb" BorderThickness="1" CornerRadius="5" Padding="15,20,15,22">
+            <TextBlock x:Name="TestName" FontSize="14" FontFamily="Consolas">
```

Comment:
Just a comment but you can render markdown text in XAML, which you can then use to write code snippets in various languages. https://docs.microsoft.com/en-us/windows/communitytoolkit/controls/markdowntextblock

Reply:
Is this only for UWP apps? I can't seem to find the markdowntextblock in the Microsoft.UI.Xaml namespace